src/AdminConsole/readme.md

```diff
@@ -25,8 +48,7 @@
     "sqlite":""
   }
 }
-
 ```
 
-## Ef migrations
+## Entity Framework (EF) migrations
 Use the ./migrate.sh shell file to create and/or apply migrations to both Sqlite and Mssql.
```

Comment:
Mention that this is optional and done automatically if you follow Get started guide